interface/main/calendar/add_edit_event.php

```diff
@@ -185,6 +200,17 @@ function DOBandEncounter()
              }
      }
     }
+    // auto create encounter fot therapy group
+    if(!empty($_POST['form_gid'])){
+                                                                                // status Took Place is the check in of therapy group
+        if ($GLOBALS['auto_create_new_encounters'] && $event_date == date('Y-m-d') && $_POST['form_apptstatus'] == '=') {
```

Comment:
The $_POST['form_apptstatus'] comes from the form_apptstatus select box in the add_edit_event form. Indeed the options listed there are from the Group Status list, but didn't think necessary to place an 'if' control in the form and change the name of the field in case the groups tab is bring used.